packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_sidecar/volumes.py

```diff
@@ -13,7 +13,7 @@
 
 
 @log_decorator(_logger, level=logging.DEBUG)
-async def save_volume_state(
+async def update_volume_status(
```

Comment:
same here. did you change the final function?
`save_volume_state` = you save some volume state.
`update_volume_status` = you update the status of a volume

Both are a bit unclear, but the second one is even more unclear.